LTTngDataExtensions/Resources/GenericEventTablePrebuiltConfigurations.json

```diff
@@ -21,6 +21,21 @@
                         }
                     ],
                   "Columns": [
+                    {
+                      "DisplayHints": {
+                        "AggregationMode": 0,
+                        "IsVisible": true,
```

Comment:
Given that most LTTng events will not have a ProviderName (except for TraceLogging) I would prefer this to either be:
a) Hidden by default
b) Dynamically hidden based on if there is any data present in column (not sure if this is possible when using .json config)

Reply:
I couldn't get dynamically hidden to work, so just defaulted this to hidden by default. I also noticed that the System Activity table views now show up (not sure if it's from this change or using a newer set of plugins), which has Provider Name & Provider visible by default for its Generic Events & Trace Logging tables.